extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -246,7 +247,12 @@ A **feature** is a specific instantiation of class containing **property values*
 
 **Statistics** provide aggregate information about the metadata. For example, statistics may include the min/max values of a numeric property for mapping property values to color ramps or the number of enum occurrences for creating histograms.
 
-By default properties do not have any inherent meaning. A property may be assigned a **semantic**, an identifier that describes how the property should be interpreted. The full list of built-in semantics can be found in the [Cesium Metadata Semantic Reference](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata/Semantics). Model authors may define their own application- or domain-specific semantics separately.
+By default properties do not have any inherent meaning. A property may be assigned a **semantic**, an identifier that describes how the property should be interpreted. Built-in semantics include `ID` and `NAME`, as defined below. Model authors may define their own application- or domain-specific semantics separately.
+
+- `ID`: Unique identifier for the feature.
+- `NAME`: Name of the feature; not required to be unique.
+
+> **TODO:** Is there any naming convention for application- or domain-specific semantics, such as an `_*` prefix?
```

Comment:
Another thing, it would be good to avoid putting TODOs in the document so we don't accidentally leave them in. an inline comment on the PR or a GH issue is how we typically like to do discussions here.

Reply:
Sounds good – I was thinking of these TODOs as being merge-blocking, I guess an inline comment would do just as well though.